contracts/InterchainTokenService.sol

```diff
@@ -901,20 +869,21 @@ contract InterchainTokenService is
         bytes32 commandId,
         string calldata sourceChain,
         string calldata sourceAddress,
-        bytes memory payload,
-        bytes32 payloadHash
-    ) internal {
+        bytes calldata payload
+    ) internal override whenNotPaused {
+        bytes32 payloadHash = keccak256(payload);
```

Comment:
the correct fix for this needs to be to inherit from a modified version of ValuedExpressExecutable which can handle the express logic. That's more work for this PR. So, let's keep the current `execute`/`executeWithToken` implementation we're using for now